src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx

```diff
@@ -47,7 +48,7 @@ export class ShowHeader extends React.Component<Props> {
           />
         )}
         <Box px={2}>
-          <ArtistNamesList artists={artists} />
+          <ArtistNamesList artists={artists} component={this} viewAllArtists={onViewAllArtistsPressed} />
```

Comment:
When working / passing components, its idiomatic to capitalize: `component` -> `Component`. The reason for this is a lowercase version used in jsx (`<component>`) breaks the compiler since it thinks its a native html element or web-component.